app/js/components/Visualizer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Visualizer = () => (
+    <div>
+        Hello, World!
```

Comment:
This will probably need to interact with the job state in some way (showing a loading icon while the job runs until it can render the contents). I don't know if you want to do that now, or later.